tests/testthat/test-copy-model-from.R

```diff
@@ -61,6 +61,21 @@ test_that("copy_from_model options work [BBR-CMF-002]", {
     stringr::str_extract(orig_mod_str, prob_pattern),
     stringr::str_extract(new_mod_str, prob_pattern)
   )
+
+
+  #Editing YAML to add star status is true from clean model
+  temp_mod_path <- create_temp_model()
+  new_mod <- read_model(temp_mod_path)
+  new_mod <- add_star(new_mod)
+
+  #Checking that star is written out to the yaml
+  model_yml <- paste0(temp_mod_path, ".yaml") %>% read_yaml() %>% names()
+  expect_identical(model_yml[5],"star")
```

Comment:
Okay as is, but testing the index 5 seems a bit more brittle than needed.  Wouldn't it be sufficient to just test that star is in name?

(Another nit-picky comment is that you do the `read_yaml` call again below, so you could just do it one time and store it as a variable.)